Processes/How-To-Submit-Content-to-the-Docs-Portal.md

```diff
@@ -1,26 +1,17 @@
 ## How To Submit Content to EOS Documentation Portal
 
-**Scenario**: You have written content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com) in .md format. Now what do you need to do?
+Once you have written your content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com), you need to submit it for review. 
+
+## Steps to Submit Content
 
 Complete the following steps to submit your content to DevRel:
 
-1. Create a PR in the DevRel repository
-   Include the following information in the ticket/issue (if applicable)
-   * Your Name
-   * Title of document
-   * Document type (template applied, if applicable)
-     * Overview
-     * Explainer
-     * Tutorial
-     * How To
-   * Who (what role) the document pertains to
-     * App Developer
-     * Node Operator
-     * Smart Contract Developer
-   * Considerations
-     * Considerations are special instructions to DevRel, such as an indication the content needs more information, request to be uploaded to the EOS Documentation portal by a specific date, …
-
-2. Upload your titled .md formatted document to the PR
-3. Submit the PR
-
-The DevRel team refines the content. The DevRel team takes into account your considerations and and will contact you if they have any questions. 
\ No newline at end of file
+1. Create a a fork of the receiving repository.
```

Comment:
a a fork -> a fork

receiving -> target

Reply:
Done

Reply:
It still reads "receiving" repo. From the standpoint of the reader it's the "target" repo. Also for consistency with step 3 which also reads target repo. @VEG-Ginna